### PR TITLE
Updated UC consent forms

### PR DESCRIPTION
**Pull request by bransteitterbr**
for https://github.com/episphere/connect/issues/694

### CODE REVIEW

File: forms/consent/UChicago_Consent_V0.04.html [hunk 1]
```diff
@@ -0,0 +1,1197 @@
...
+                style="width:211pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt">
+                <p class="s1" style="padding-top: 6pt;padding-left: 5pt;text-indent: 0pt;text-align: left;">Connect
+                    Support Center</p>
+            </td>
+            <td
+                style="width:257pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt">
+                <p class="s1"
+                    style="padding-left: 5pt;padding-right: 85pt;text-indent: 0pt;line-height: 20pt;text-align: left;">
+                    <a href="mailto:ConnectSupport@norc.org" class="s2" target="_blank">MyConnect.cancer.gov/support
+                        Email: ConnectSupport@norc.or</a>g Phone: 1-877-505-0253</p>
```
Comment (JoeArmani) [inline forms/consent/UChicago_Consent_V0.04.html:248]:
```suggestion
                        Email: ConnectSupport@norc.org</a> Phone: 1-877-505-0253</p>
```

File: forms/consent/UChicago_Consent_V0.04.html [hunk 2]
```diff
@@ -0,0 +1,1197 @@
...
+            <p style="text-indent: 0pt;text-align: left;"><br /></p>
+            <h1 style="padding-top: 4pt;padding-left: 20pt;text-indent: 0pt;text-align: left;">IT IS YOUR CHOICE TO TAKE
+                PART IN THE STUDY</h1>
+            <p style="padding-top: 6pt;padding-left: 20pt;text-indent: 0pt;text-align: left;">You may choose not to take
+                part in this study for any reason. If you join this study, you may change your mind and stop
+                participating in the study at any time and for any reason. Deciding not to participate in the study will
+                not affect your routine medical care, your relationship with your current medical provider, or your
+                health insurance. If you do choose to leave the study, please contact the Connect Support Center
+                (Cancer.gov/connectstudy/support). You can choose to stop getting study messages, stop doing study
+                activities, or to withdraw from (quit)Connect altogether.</p>
```
Comment (JoeArmani) [inline forms/consent/UChicago_Consent_V0.04.html:543]:
```suggestion
                activities, or to withdraw from (quit) Connect altogether.</p>
```

File: forms/consent/UChicago_Consent_V0.04.html [hunk 3]
```diff
@@ -0,0 +1,1197 @@
...
+        <li data-list-text="■">
+            <p style="padding-left: 56pt;text-indent: -18pt;line-height: 15pt;text-align: left;">Donate your first blood
+                specimen</p>
+        </li>
+    </ul>
+    <p style="padding-top: 5pt;padding-left: 20pt;text-indent: 0pt;text-align: left;">If you are unable to finish these
+        study activities, you will not receive payment for the parts you completed. We expect most people to complete
+        this first set of activities shortly after agreeing to</p>
+    <p style="text-indent: 0pt;text-align: left;"><br /></p>
+    <p style="padding-top: 4pt;padding-left: 20pt;text-indent: 0pt;text-align: left;">join Connect. Payment is provided
```
Comment (JoeArmani) [inline forms/consent/UChicago_Consent_V0.04.html:1019]: Is this sentence break intended?

**Review by JoeArmani — APPROVED** [review]
looks good, minor grammatical suggestions/comments.

File: forms/consent/UChicago_Consent_V0.04.html [hunk 4]
```diff
@@ -0,0 +1,1195 @@
...
+            <p style="padding-top: 5pt;padding-left: 56pt;text-indent: -18pt;text-align: left;">You will probably not
+                benefit directly from joining Connect, but your information could help prevent cancer and other diseases
+                in the future.</p>
+        </li>
+        <li data-list-text="■">
+            <p style="padding-top: 5pt;padding-left: 56pt;text-indent: -18pt;text-align: left;">More details can be
+                found below or on the Connect website (Cancer.gov/connectstudy).</p>
+            <h1 style="padding-top: 5pt;padding-left: 11pt;text-indent: 0pt;text-align: left;">If you agree to join
+                Connect, we will ask you to:</h1>
+            <p style="text-indent: 0pt;text-align: left;"><br /></p>
```
Comment (we-ai) [inline forms/consent/UChicago_Consent_V0.04.html:329]:
```suggestion
```

File: forms/consent/UChicago_Consent_V0.04.html [hunk 5]
```diff
@@ -0,0 +1,1195 @@
...
+                            When: <span class="s1">When you join and at least once a year in the future.</span></p>
+                        <p class="s3" style="padding-top: 6pt;padding-left: 5pt;text-indent: 0pt;text-align: left;">How:
+                            <span class="s1">You can complete surveys online with a computer, tablet, or mobile
+                                phone.</span></p>
+                    </td>
+                </tr>
+                <tr style="height:74pt">
+                    <td
+                        style="width:66pt;border-top-style:solid;border-top-width:1pt;border-left-style:solid;border-left-width:1pt;border-bottom-style:solid;border-bottom-width:1pt;border-right-style:solid;border-right-width:1pt">
+                        <p style="text-indent: 0pt;text-align: left;"><br /></p>
```
Comment (we-ai) [inline forms/consent/UChicago_Consent_V0.04.html:360]: ?

**Comment by we-ai** [comment]
Does participant names align well in re-rendered pdf file of this version?

**Pull request merged**